docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -38,15 +38,16 @@ You can use a short name - *browser alias* - to identify these browsers when lau
 
 The following table lists browsers that can be detected automatically.
 
-Browser           | Browser Alias
------------------ | -------------------
-Chromium          | `chromium`
-Google Chrome     | `chrome`
-Internet Explorer | `ie`
-Microsoft Edge    | `edge`
-Mozilla Firefox   | `firefox`
-Opera             | `opera`
-Safari            | `safari`
+Browser                                      | Browser Alias
+---------------------------------------------| -------------------
+Chromium                                     | `chromium`
+Google Chrome (Regular Channels)             | `chrome`
```

Comment:
I suppose it should be just `Google Chrome`. I don't see a `regular` word in their page: https://www.chromium.org/getting-involved/dev-channel